html/semantics/interactive-elements/the-dialog-element/centering.html

```diff
@@ -34,6 +34,19 @@
 testDialogCentering("horizontal-tb", "vertical-rl", "horizontal-tb", "tall viewport", 40, 100, "right", 40 / 2 - DIALOG_WIDTH / 2);
 testDialogCentering("vertical-rl", "horizontal-tb", "vertical-rl", "tall viewport", 40, 100, "top", 100 / 2 - DIALOG_HEIGHT / 2);
 
+function getDialogWritingMode(writingMode, containerWritingMode, dialogWritingMode) {
+  if (dialogWritingMode) {
```

Comment:
FWIW this could be rewritten as `return dialogWritingMode || containerWritingMode || writingMode || "horizontal-tb"`. Maybe you don't even need a separate function at that point.